### PR TITLE
resolve issue #91

### PR DESCRIPTION
**Pull request by mamund**
resolve issue #91

### CODE REVIEW

File: draft-06/draft-06.txt [hunk 1]
```diff
@@ -506,6 +506,13 @@ Amundsen, et al.          Expires July 17, 2021                 [Page 9]
 Internet-Draft     Application-Level Profile Semantics      January 2021
 
 
+1.4.  Identifying an ALPS Document
+
+   An ALPS vocabulary is identified by a unique URL.  This URL SHOULD be
+   dereferencable.  All ALPS URLs MUST be unique and all ALPS documents
```
Comment (malevy) [inline draft-06/draft-06.txt:512]: is "dereferencable" a word? Merriam-Webster doesn't think so ;-)
what is the intention of this sentence? Does it mean that the URL should resolve to the document that contains the ALPS vocabulary?

Comment (mamund) [reply draft-06/draft-06.txt:512]: https://www.yourdictionary.com/dereferenceable

essentially a mis-spelling here.

can rewrite to ...

"This SHOULD be a URL that can be dereferenced.."

good?

Comment (malevy) [reply draft-06/draft-06.txt:512]: yep!

Comment (mamund) [reply draft-06/draft-06.txt:512]: excellent. 

thanks for the careful read. please keep them coming!

Comment (malevy) [reply draft-06/draft-06.txt:512]: I saw "dereferencable" in a couple of spots but only questioned the one.

Comment (mamund) [reply draft-06/draft-06.txt:512]: got it. will root them out<g>

Comment (mamund) [reply draft-06/draft-06.txt:512]: expunged!

**Pull request merged**